src/store-engine.js

```diff
@@ -25,11 +25,11 @@ var storeAPI = {
 
 	// set will store the given value at key and returns value.
 	// Calling set with value === undefined is equivalent to calling remove.
-	set: function(key, value) {
+	set: function(key, value, isSecure = false) {
```

Comment:
Using default parameter values will break compatibility with a bunch of browsers that Store.js is supposed to support.

Reply:
what about something like this: 

`set: function(key, value, options) { ...`

`this.storage.write(this._namespacePrefix + key, this._serialize(value), options)` 

and in cookieStorage.js: 
function write(key, data, isSecure = false) {
	if(!key) { return }
	var opts = options ? options : ""
	doc.cookie = escape(key) + "=" + escape(data) + "; expires=Tue, 19 Jan 2038 03:14:07 GMT; path=/" + options
}	}` 

the user could then set a cookie like: 
`store.set('myName', 'data', ';Secure;SameSite=strict')`